scripts/fetch-docs.js

```diff
@@ -1,17 +1,22 @@
 const path = require('path');
 const fs = require('fs');
 const fetch = require('node-fetch');
+const TurndownService = require('turndown');
+require('dotenv').config();
 
 const logger = require('./utils/logger');
 const createIndexPages = require('./create-index-pages');
 const { BASE_URL, BASE_DIR } = require('./constants');
+const { formatWhitespace, escapeBraces } = require('./codemods/replaceChars');
```

Comment:
I think we can remove this `require` (and possibly these files). It doesn't look like we are using them.

```suggestion
```

Reply:
Yep, wasn't sure about removing files altogether, since we may need to further transform. I'll remove files and require now for time being